### ISSUE TITLE
DATA no change: 3/22/20 and 3/23/20 in time_series_19-covid-Confirmed.csv

### ISSUE THREAD

**Issue by jaalto** [CSSEGISandData/COVID-19#1403]
Please check. There is no difference in number of cases in rows in 3/22/20	and 3/23/20 in file time_series_19-covid-Confirmed.csv

**Comment by cheetobandito** [comment]
It would appear the column heading changed for Country in the daily reports.  Maybe this broke a piece of automation that was generating the time series csv file.

**Comment by jkellndorfer** [comment]
Yes, noticed the same, 2020-03-23 data are a duplicate of 2020-03-22 with difference that 2020-03-23 has missing values 
Example Record:
"New York County, NY",US,40.7128,-74.006,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,4,11,11,12,19,0,0,0,0,0,0,0,0,0,0,0,0,0,

**Comment by charlesg** [comment]
Same here, numbers are the same as yesterday... Following this issue so I know when to re-download updated data.

**Comment by chartzngrafs** [comment]
Would be great if someone would fix this... time series data is unreliable as it stands.

**Comment by Hikmer** [comment]
It is very disappointing that this data set isn't being managed in a reliable way.  The world is in crisis and panic, looking at these reports but the lack of data integrity is astounding.  Is someone getting paid to do this effort or is just a best effort situation?

**Comment by Hikmer** [comment]
They are making changes to metadata in mid-crisis.  It is rather difficult to create anything reliable that doesn't require going back and fixing things.

**Comment by maisonml** [comment]
They have mentioned in csse_covid_19_time_series/README.md that they will no longer update those 3 files going forward. Instead please use time_series_covid19_confirmed_global.csv.

**Comment by Hikmer** [comment]
Wow thats disappointing, many did a lot of work off of those files.  Unbelievable.

**Comment by jfhirsch** [comment]
New files (time_series_covid19_confirmed_global.csv,  time_series_covid19_deaths_global.csv) do not include separate entries for each state in the US.   Will those be maintained in a separate file?
